docs/developer/explanation/debian-package-publishing.rst

```diff
@@ -0,0 +1,104 @@
+Debian package publishing
+=========================
+
+The Debian package lifecycle describes the process from a `source package upload <https://documentation.ubuntu.com/project/how-ubuntu-is-made/concepts/package-format/#source-packages>`_
+to the `Ubuntu package archive <https://documentation.ubuntu.com/project/how-ubuntu-is-made/concepts/package-archive/>`_.
+This involves multiple steps in which the original source package is verified,
+built into a `binary package <https://documentation.ubuntu.com/project/how-ubuntu-is-made/concepts/package-format/#binary-packages>`_
+and finally published into the package archive.
+
+Please note that this document is not describing the policies and
+review processes involved in accepting package changes
+or the migrations between suites, but rather the technical steps
+involved in the publishing process.
+
+#####################
+Source package upload
+#####################
+
+The first step of the publishing process is to upload a source package
+with a signed source ``.changes`` file. The signed document references
+and includes checksums of the files to upload and the source package.
+These files are usually uploaded with ``dput`` or ``dput-ng``, which use
+FTP or SFTP to push the files to Launchpad.
```

Comment:
It is worth explaining here that the uploads are sent to `upload.ubuntu.com` and that the Ubuntu upload processor runs there.